WebApiBackend/WebApiBackend/Controllers/UserController.cs

```diff
@@ -381,5 +382,24 @@ private void SendResetEmail(string destination, string resetToken)
             smtpClient.EnableSsl = true;
             smtpClient.Send(mailMessage);
         }
+
+        /// <summary>
+        /// GET Method - Get the id of the current user
+        /// </summary>
+        /// <response code="200">Success</response>
+        /// <response code="401">Not an authorised user</response>
+        /// <returns>Retrieves the id of the current user</returns>
+        [HttpGet("getUserID")]
+        [Authorize]
+        public ActionResult<int> GetUserID()
+        {
+            var identity = HttpContext.User.Identity as ClaimsIdentity;
```

Comment:
Maybe put a try catch around the statement, so that if a someone tries to call the end point without being logged in or having a cookie, the whole server doesn't crash.

Reply:
Done (: